app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -54,16 +54,19 @@ def start
       :pod_namespace   => namespace
     ))
 
-    pod = pod_definition
-    _log.info("creating pod #{pod_full_name} to analyze docker image " \
-              "#{options[:docker_image_id]}: #{pod.to_json}")
-
+    log_pref = nil
     begin
+      log_pref = "Getting Service Account for pod [#{pod_full_name}]"
+      _log.info(log_pref)
+      inspector_admin_secret_name = inspector_admin_secret
+      log_pref = "Creating pod [#{pod_full_name}]"
+      pod = pod_definition(inspector_admin_secret_name)
+      _log.info("#{log_pref} to analyze docker image [#{options[:docker_image_id]}] [#{pod.to_json}]")
```

Comment:
@moolitayer we'll have to discuss this block (mtng?) because ATM it seems to be misleading and far from being something that we can merge.